addons/survey/views/survey_question_views.xml

```diff
@@ -167,40 +180,41 @@
                             <field name="description" widget="html"/>
                         </page>
                         <page string="Options" name="options" attrs="{'invisible': [('is_page', '=', True)]}">
-                            <group string="Constraints">
-                                <group>
-                                    <field name="constr_mandatory" string="Mandatory Answer"/>
-                                    <field name="constr_error_msg" attrs="{'invisible': [('constr_mandatory', '=', False)]}"/>
+                            <group>
+                                <group string="Answers">
+                                    <field name="save_as_email" string="Input must be an email"/>
+                                    <field name="save_as_nickname"/>
+                                    <field name="validation_required"/>
+                                    <field name="column_nb" string="Number of columns" attrs="{'invisible':[('question_type','not in',['simple_choice', 'multiple_choice'])]}"/>
```

Comment:
We usually keep one space between every arguments.

So 
`{'invisible':[('question_type','not in',['simple_choice', 'multiple_choice'])]}`

Becomes

`{'invisible':[('question_type', 'not in', ['simple_choice', 'multiple_choice'])]}`